spec/controllers/resource_controller_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe ResourceController, type: :controller do
```

Comment:
I changed the routes.rb
I don't know if it's the right solution. My repo is not able to execute ./bin/rspec because of wrong port settings. I'm not good enough in the rails development to fix it.
@kevin-klein Did you see incorrect routes for /resource/ ?